webview-ui/src/components/browser/BrowserSettingsMenu.tsx

```diff
@@ -162,32 +161,50 @@ export const BrowserSettingsMenu = () => {
 			</VSCodeButton>
 
 			{showInfoPopover && (
-				<InfoPopover ref={popoverRef}>
+				// InfoPopover - Dropdown container with connection details
+				<div
+					ref={popoverRef}
+					className="absolute top-[30px] right-0 z-[100] w-[60dvw] max-w-[250px] rounded p-2.5 shadow-lg"
+					style={{
+						backgroundColor: "var(--vscode-editorWidget-background)",
+						border: "1px solid var(--vscode-widget-border)",
+						boxShadow: "0 2px 8px rgba(0, 0, 0, 0.15)",
+					}}>
```

Comment:
I think the shadow-lf you added covered the boxShadow style already, so we can just move the backgroundColor and border to tailwind className so we can remove the style block:
```suggestion
					className="absolute top-[30px] right-0 z-[100] w-[60dvw] max-w-[250px] rounded p-2.5 shadow-lg bg-[var(--vscode-editorWidget-background)] border-[var(--vscode-widget-border)] border-1">
```

If you don't think shadow-lg works, we can use shadow-[0_2px_8px_rgba(0,0,0,0.15)] ?